include/functions.h

```diff
@@ -711,12 +711,12 @@ void func_800B42F8(Actor* actor, Lights* mapper, GlobalContext* globalCtx);
 void func_800B4AEC(GlobalContext* globalCtx, Actor* actor, f32 param_3);
 void func_800B4B50(Actor* actor, s32 iParm2, GlobalContext* globalCtx);
 void func_800B4EDC(GlobalContext* globalCtx, Vec3f* pzParm2, Vec3f* pzParm3, f32* pfParm4);
-// void func_800B4F40(TargetContext* targetContext, s32 param_2, UNK_TYPE4 param_3, UNK_TYPE4 param_4, UNK_TYPE4 param_5);
-void func_800B4F78(TargetContext* targetContext, u8 type, GlobalContext* globalCtx);
-void func_800B5040(TargetContext* targetContext, Actor* actor, u8 type, GlobalContext* globalCtx);
+// void func_800B4F40(TargetContext* targetCtx, s32 param_2, UNK_TYPE4 param_3, UNK_TYPE4 param_4, UNK_TYPE4 param_5);
+void func_800B4F78(TargetContext* targetCtx, u8 type, GlobalContext* globalCtx);
+void func_800B5040(TargetContext* targetCtx, Actor* actor, u8 type, GlobalContext* globalCtx);
 void Actor_TargetContextInit(TargetContext* targetCtx, Actor* actor, GlobalContext* globalCtx);
 void func_800B5208(TargetContext* targetCtx, GlobalContext* globalCtx);
-void func_800B5814(TargetContext* targetContext, Player* player, Actor* param_3, GlobalContext* globalCtx);
+void func_800B5814(TargetContext* targetCtx, Player* player, Actor* param_3, GlobalContext* globalCtx);
```

Comment:
Minor: Since we know `param_3` is an `Actor*`, do we want to just call it `actor`.